internal/otelcollector/config/log/agent/testdata/config.yaml

```diff
@@ -92,6 +92,26 @@ receivers:
               type: severity_parser
               if: attributes.level != nil
               parse_from: attributes.level
+            - id: trace-parent-parser
+              type: regex_parser
+              if: attributes.trace_id == nil
+              parse_from: attributes.traceparent
+              regex: ^[0-9a-f]{2}-(?P<trace_id>[0-9a-f]{32})-(?P<span_id>[0-9a-f]{16})-(?P<trace_flags>[0-9a-f]{2})$
+              trace:
+                trace_id:
+                    parse_from: attributes.trace_id
```

Comment:
Why the intendation is bigger here?

Reply:
This is how the go yaml marshaller default indentation.

Reply:
In the operator with id: trace-parser below, there you don't have that problem.. sure that it is not fixable?

Reply:
it's fixable with the marshaller configuration; we can configure it with the desired indentation. But let us do it separately from this PR since this will change all the test and config generation